translations/pt-BR/content/actions/guides/building-and-testing-java-with-gradle.md

```diff
@@ -38,22 +38,22 @@ Você também pode adicionar este fluxo de trabalho manualmente, criando um novo
 
 {% raw %}
 ```yaml
-name: Java CI
+nome: Java CI
 
-on: [push]
+em: [push]
```

Comment:
They appear to be translating elements in the YAML

Reply:
@chiedo maybe the revert is bringing in broken examples

Reply:
#302